x-pack/plugins/fleet/server/routes/epm/handlers.ts

```diff
@@ -66,7 +66,7 @@ const CACHE_CONTROL_10_MINUTES_HEADER: HttpResponseOptions['headers'] = {
 export const getCategoriesHandler: FleetRequestHandler<
   undefined,
   TypeOf<typeof GetCategoriesRequestSchema.query>
-> = async (context, request, response) => {
+> = async (_context, request, response) => {
```

Comment:
can this be reverted?

Reply:
Sure I'll change it back if you like. It's not being used though so I renamed it for clarity.